src/app/zap-templates/zcl/data-model/chip/laundry-dryer-controls-cluster.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<configurator>
+    <domain name="CHIP" />
+
+    <enum name="DrynessLevelEnum" type="enum8">
```

Comment:
This is not what the spec has, as far as I can tell.  What exact spec version is this targeting, if it's not master or 0.7 ballot?  That needs to be clearly stated in the PR for this to be reviewable.

Reply:
This is based on an updated version of the spec that is still in PR:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7766